api/v1alpha1/virtualmachineimage_conversion.go

```diff
@@ -93,6 +93,10 @@ func convert_v1alpha1_VirtualMachineImage_OVFEnv_To_v1alpha2_VirtualMachineImage
 
 func convert_v1alpha2_VirtualMachineImageStatusConditions_To_v1alpha1_VirtualMachineImageStatusConditions(
 	conditions []metav1.Condition) []Condition {
+	if conditions == nil || len(conditions) == 0 {
```

Comment:
This can be simplified to just `len(conditions) == 0` since that covers `nil` too. Unless something complains or really needs an empty slic, I'd return `nil` here instead.

Reply:
Done